maven/api-maven/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/strategy/DefaultTransitiveExclusionPolicy.java

```diff
@@ -19,7 +19,7 @@
 import org.jboss.shrinkwrap.resolver.api.maven.ScopeType;
 
 /**
- * Applies the default behavior exhibited by Maven with regards to handling transitive dependencies during resolution.
+ * Applies the default behavior exhibited by Maven in regard to handling transitive dependencies during resolution.
```

Comment:
What about "Applies the default behavior exhibited by Maven in handling transitive dependencies during resolution."